editor/editor_properties.cpp

```diff
@@ -2036,7 +2036,7 @@ void EditorPropertyResource::_menu_option(int p_which) {
 			ERR_BREAK(!obj);
 			Resource *resp = Object::cast_to<Resource>(obj);
 			ERR_BREAK(!resp);
-			if (get_edited_object() && base_type != String() && base_type == "Script") {
+			if (get_edited_object() && base_type != String() && base_type == "Script" && !res.is_null()) {
 				//make visual script the right type
 				res->call("set_instance_base_type", get_edited_object()->get_class());
```

Comment:
I wonder if this call is not meant to be made on `resp` as just defined/safety-checked instead of `res`?

Reply:
yes, proper fix is likely replacing 
```
res->call
by
resp->call
```

Reply:
I'll test that, can't believe I missed that. Sorry!